src/lib.rs

```diff
@@ -17,15 +17,12 @@
     const_err,
     dead_code,
     improper_ctypes,
-    legacy_directory_ownership,
     non_shorthand_field_patterns,
     no_mangle_generic_items,
     overflowing_literals,
     path_statements,
     patterns_in_fns_without_body,
-    plugin_as_library,
```

Comment:
What's with these ones?

Reply:
They have been removed as rustc lints and are hard errors now. When adding them there is a warning.